src/TraceEvent/Parsers/ClrTraceEventParser.cs

```diff
@@ -5335,10 +5399,12 @@ public override object PayloadValue(int index)
             switch (index)
             {
                 case 0:
-                    return TypeID;
+                    return TypeName;
```

Comment:
It should just see no ID->Name mappings and thus TypeName will be "" for everything.  
However just to be super-safe I have added a change to bump the version number on the ETLX file which will force it to be re-created in PerfView.